OWML.Common/Menus/IModMenu.cs

```diff
@@ -13,8 +13,10 @@ public interface IModMenu
         List<IModButton> Buttons { get; }
         List<IModTitleButton> TitleButtons { get; }
         List<IModLayoutButton> LayoutButtons { get; }
+        List<IModPromptButton> PromptButtons { get; }
 
-        IModTitleButton GetButton(string title);
```

Comment:
Breaking change, some mods might use this...

Reply:
You could add GetButton again, mark it deprecated ([Obsolete]) and make it call GetTitleButton

Reply:
K